rllib/algorithms/a2c/a2c.py

```diff
@@ -127,6 +127,12 @@ def validate_config(self, config: AlgorithmConfigDict) -> None:
                     "`microbatch_size` (try setting them to the same value)! "
                     "Otherwise, microbatches of desired size won't be achievable."
                 )
+        if config.get("num_gpus", 0) > 1:
+            raise ValueError(
+                "A2C with gpu learning enabled only supports 1 "
+                "gpu. Currently, the methods by which mini-batch"
+                " gradients are computed doesn't support >1 gpus"
```

Comment:
move whitespace to the end of last line man.
also, this still doesn't tell users why the method doesn't support >1 gpus.
I feel like it's not really "can't be supported" right? it's "there is no need.".

Reply:
the linter did most of the filtering on its own here

Reply:
I'm just gonna say is not current'y supported. We can figure out later why its the case we can't do batch gradients across multiple gpus.

Reply:
use gpu to just batch and average gradients feels like a waste.
but sure.